posts/2022-04-15-libertydevops-gitjenkinshelmiks.adoc

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: "Continuous integration and delivery (CI/CD) of cloud-native Java applications using Jenkins"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/shamjithantholi
+author_github: https://github.com/shamjithantholi
+seo-title: TITLE - OpenLiberty.io
+seo-description: DESCRIPTION
```

Comment:
Replace DESCRIPTION with "How to build a DevOps pipeline for a cloud-native Java application with Jenkins as the core controller component."

Reply:
Done